portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -22,12 +22,14 @@
   private final String email;
   private final String message;
   private final long timestamp;
+  private final double score;
 
-  public Comment(long id, String name, String email, String message, long timestamp) {
+  public Comment(long id, String name, String email, String message, long timestamp, double score) {
```

Comment:
when naming, better to specify which score it is, add javadoc for public class and methods, explain what score this is, what it means when it is high/low, what is the expected range of the score, etc.